src/constants/ErrResponses.ts

```diff
@@ -0,0 +1,5 @@
+export const GENERIC_400_MSG = "The request object received had invalid syntax or missing information.";
+export const USER_NOT_WHITELISTED = "Cannot create an account since the email id provided does not have any affiliation with HKN. Visit hkn.ucsd.edu to know more.";
+export const USER_ALREADY_EXISTS = "The provided email is aready linked to an account. Please check the email you have entered.";
+export const FIREBASE_AUTH_CREATE_USER_ERR = "There was an internal error creating an auth account. Please try again later.";
+export const FIRESTORE_UPDATE_DOC_ERR = "There was an error modifying the user document. Please try again later."
```

Comment:
Okay. I guess the last 2 are internal errors that probably need not be mentioned to the user. For the rest I think it is fine since we will eventually display some error message or prompt to the user when signup fails right?

Reply:
:+1: